Coolbooru/Properties/AssemblyInfo.cs

```diff
@@ -31,5 +31,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.0")]
-[assembly: AssemblyFileVersion("2.0")]
+[assembly: AssemblyVersion("2.0.*")]
```

Comment:
Not sure if this change is actually necessary. Also not very familiar with NuGet, what will this be filled in with when we need to publish a new version?